test/views/helpers/LitresSpecHelper.scala

```diff
@@ -236,4 +237,14 @@ trait LitresSpecHelper extends ViewSpecHelper {
       }
     }
   }
+
+  def validateCreditClaim(document: Document): Unit = {
+    val insetText = document.getElementsByClass(Selectors.insetText)
+    val claimCreditNote = "Returns ending after 1 April 2025 cannot be used to claim credit for levy reported or paid in returns ending before 1 April 2025. Notify HMRC separately to claim credits for the levy accounted for or paid in returns ending on or before 31 March 2025."
```

Comment:
We don't need to duplicate the text here just to avoid the HTML content, in order for us to verify with insetText. We can make use of `Jsoup` for that. For ex: above could be written as following to avoid duplication of content and should still work

`val claimCreditNote = Jsoup.parse(Messages("creditClaims.info", "")).text()`

Also, reason for asking to inline the link aswell is so that we test the link too by accessing `insetText.get(0).getElementsByClass("govuk-link")`. That also allows our message access to simply then be `Messages("creditClaims.info")` - because we know this is a static link and can be inlined